examples/apps/spaces/tsconfig.json

```diff
@@ -1,14 +1,12 @@
 {
     "extends": "@fluidframework/build-common/ts-common-config.json",
-    "exclude": [
-        "dist",
-        "node_modules"
-    ],
+    "exclude": ["dist", "node_modules"],
     "compilerOptions": {
         "target": "es6",
         "declarationDir": "./dist",
         "outDir": "./dist",
         "jsx": "react",
+        "skipLibCheck": true,
         "types": [
```

Comment:
Please add comment for each of these.

Reply:
That isn't wrong but it isn't as helpful as it could be. Would also be useful to know where errors come from. Like observed errors in @foo/bar. Then future readers would have a better idea when to revisit.
The common line across them all does help find them. Thanks.